.github/workflows/cd.yml

```
@@ -0,0 +1,131 @@
+              docker compose -f docker-compose-prod.yml down || true
+              docker compose -f docker-compose-prod.yml build
+              docker compose -f docker-compose-prod.yml up -d
+              
+              sleep 30
+            else
+              echo "⚡ Code-only changes - ultra fast deployment"
+              
+              # Update code first
+              echo "📥 Updating code..."
+              git pull origin cd-settings
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)